share/picongpu/pypicongpu/examples/laser_wakefield/main.py

```diff
@@ -0,0 +1,208 @@
+"""
+This file is part of PIConGPU.
+Copyright 2024 PIConGPU contributors
+Authors: Masoud Afshari, Brian Edward Marre
+License: GPLv3+
+"""
+
+from picongpu import picmi
+from picongpu import pypicongpu
+import numpy as np
+
+"""
+@file PICMI user script reproducing the PIConGPU LWFA example
+
+This Python script is example PICMI user script reproducing the LaserWakefield example setup, based on 8.cfg.
+"""
+
+# generation modifiers
+ENABLE_IONS = True
```

Comment:
If this suppose to reproduce the LWFA example case 0, please deactivate ions and ionization.

Reply:
No the default is supposed to represent the case with ions.

Since we use the LWFA example to test the Ionizer configuration we may not disable ionization in the example, which in turn requires ions.